etc/kernel-launchers/kubernetes/scripts/launch_custom_resource.py

```diff
@@ -48,15 +48,15 @@ def launch_custom_resource_kernel(kernel_id, port_range, response_addr, public_k
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument('--RemoteProcessProxy.kernel-id', dest='kernel_id', nargs='?',
+    parser.add_argument('--kernel-id', '--RemoteProcessProxy.port-range', dest='kernel_id', nargs='?',
```

Comment:
A small typo above, should be `--RemoteProcessProxy.kernel-id`

Reply:
thanks for catching it, fixed it